src/encoded/commands/export_data.py

```diff
@@ -0,0 +1,102 @@
+from pyramid import paster
+from urllib.parse import urlparse
+from multiprocessing.pool import ThreadPool
+import requests, logging, json
+
+EPILOG = __doc__
+
+'''
+Use this here tool to export something like a collection of Pages to inserts.
+This tool be wicked flexible so for best results, make sure to craft a good search query, otherwise you'll crash things if you try to export
+something like /search/?type=All&limit=all (seriously, this command does NOT use streaming, it WILL load things into RAM).
+
+Recommended examples:
+  - bin/export-data "https://data.4dnucleome.org/search/?type=Page&limit=all" -u ACCESS_KEY_ID -p ACCESS_KEY_SECRET > new_page_inserts_file.json
+  - bin/export-data "https://data.4dnucleome.org/search/?type=StaticSection&limit=all" -u ACCESS_KEY_ID -p ACCESS_KEY_SECRET > new_static_section_inserts_file.json
+
+This has not been tested, but theoretically should be able to pipe inserts from export_data to import_data (e.g. transfer data to different server) for __small__ amounts of data (due to lack of streaming).
+'''
+
+
+def run(search_url, username='', password=''):
+
+    url = urlparse(search_url)
+
+    if url.scheme not in ('http', 'https') or '/search/?type=' not in search_url:
```

Comment:
Might it be better to check for "/search/" and "type=" in url separately? Query param order is arbitrary